src/segger/__init__.py

```diff
@@ -1,8 +1,14 @@
-__all__ = ["data", "models", "prediction", "training"]
+import importlib.util
+
+cupy_available = importlib.util.find_spec("cupy") is not None
 
+from segger.data import *
```

Comment:
In my experience, using import paths relative to roots is more readable and robust than using relative imports.

Reply:
No need, `importlib` is builtin with Python.